setup.py

```diff
@@ -35,5 +38,6 @@ def readme():
     packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
     include_package_data=True,
     install_requires=required,
-    extras_require={"full": optional_required,},
+    extras_require={"full": optional_required,
+                    "test": test_required},
```

Comment:
Can test be both optional and test requirements? Same for nightly

Reply:
Hey @Yard1 i am not sure if I understand your comment correctly, the requirements-test.txt is now being installed for running the checks. And that's done by modifying the test.yaml file. Are you looking for something else?

Reply:
Something like this:
```suggestion
                    "test": test_required + optional_required},
```